### DISCUSSION TITLE
Upgrade installer breaks shortcut / taskbar pin

### DISCUSSION THREAD

**Discussion by albegouws** [TabularEditor/TabularEditor3#1041] [Ideas]
**Describe the issue**
I pin the app to my taskbar.  Whenever a new version is installed, the taskbar icon breaks and you need to recreate.  This is really not something major, just an annoyance. 

** Which version are you currently using? **
3.9.0 but problem has existed from the beginning.

**To Reproduce**
Are you connected to SSAS/AAS/Power BI or working offline?
N/A
1: Open App.
2: Right click task bar icon and Pin
3: Run installer to upgrade to new version
4: Pin is now broken and needs to be recreated.

**Expected behavior**
Upgrades should not break the pin.

**Screenshots**

**Desktop (please complete the following information):**
 - Windows version: 11

**Additional context**

**Comment by DBojsen** [comment]
Hi @albegouws 

I agree that it is annoying.
We already have on our backlog to improve the upgrade experience including this, but added your vote to the feature.

Best Regards

David - Tabular Editor 3 Support
